bemani/utils/psmap.py

```diff
@@ -67,9 +73,7 @@ def read_string(offset: int) -> str:
         except ValueError:
             pass
 
-        if nodetype == 0x00:
```

Comment:
Do we encounter node type 0 in 64-bit applications?

Reply:
I removed this line because when I was using this to try to reverse some psmaps it would throw an exception. I forget the exact example that broke it.

Reply:
Okay, but its needed then. Fair enough.